BUILD

```diff
@@ -17,3 +17,9 @@ gazelle_test(
     size = "small",
     workspace = "//:BUILD",
 )
+
+filegroup(
+    name = "clang_tidy_config",
+    srcs = [".clang-tidy"],
+    visibility = ["//visibility:public"],
+)
```

Comment:
What about our strict vs test configs that we have setup?

Reply:
i dunno how to do that yet. so far it doesn't even seem like this is actually the effective .clang-tidy file, despite being exactly like the examples. still trying to debug that.

Reply:
oh it seems that it's just not picking up custom .clang-tidy for individual targets (//... works). https://github.com/erenon/bazel_clang_tidy/issues/69. The first one should work for us in CI for now.